configuration.yaml

```diff
@@ -16,11 +16,6 @@ variables:
         - Addresses for the TPU services to be used by the XRT client. ";"
           separated list of addresses, e.g. localservice;0;localhost:51011.
       type: string
-    XRT_WORKERS:
-      description:
-        - Addresses for the XRT workers to be used by the XRT client, for
-          example localservice:0;grpc://localhost:51011
-      type: string
     XRT_DEVICE_MAP:
```

Comment:
Why is this flag around?

Reply:
will delete later as a task in clean XRT in Migrate to pull XLA from OpenXLA

Reply:
will delete in Will's PR: https://github.com/pytorch/xla/pull/5240